Blockzilla/SettingsViewController.swift

```diff
@@ -280,13 +276,15 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         tableView.allowsSelection = true
         tableView.estimatedRowHeight = 44
 
-        toggles = initialToggles
-        for (i, blockerToggle) in toggles {
-            let toggle = blockerToggle.toggle
-            toggle.onTintColor = UIConstants.colors.toggleOn
-            toggle.addTarget(self, action: #selector(toggleSwitched(_:)), for: .valueChanged)
-            toggle.isOn = Settings.getToggle(blockerToggle.setting)
-            toggles[i] = blockerToggle
+        initializeToggles()
+        for (sectionIndex, toggleArray) in toggles{
+            for (cellIndex, blockerToggle) in toggleArray{
```

Comment:
nit: whitespace, can we make sure every curly brace has a space before it as is in the rest of the file?